src/operator/account/AccountView.tsx

```diff
@@ -15,11 +15,19 @@ import PageSpinner from 'src/perf/components/PageSpinner'
 const AccountView: FC = () => {
   const {account, accountStatus} = useContext(AccountContext)
 
+  let accountTitle = ''
+  if (account?.name) {
+    const accountIdPart = account.id ? `(${account.id})` : ''
+    accountTitle = `${account.name} ${accountIdPart} `
+  } else {
+    accountTitle = `Account ${account?.id}`
```

Comment:
two questions: why not just set `accountTitle` to `Account ${account?.id}` on line 18 and remove this extra `else` statement? that way three lines and some extra syntactic parts are removed.

what happens if `id` doesn't exist? will it just say `Account `? is it even possible for `account.id` not to exist?

Reply:
*if* account is there, then the id will be there, but the account isn't there initially.

because the 'account' variable isn't set yet when the page first loads.  so either need the code above, or something like this:

``` let accountTitle = 'loading....'
  if (account) {
    if (account?.name) {
      accountTitle = `${account.name} (${account.id})`
    } else {
      accountTitle = `Account ${account.id}`
    }
  } ```